tests/gis_tests/geoapp/test_functions.py

```diff
@@ -442,6 +442,30 @@ def test_isempty(self):
         )
         self.assertSequenceEqual(City.objects.filter(point__isempty=True), [empty])
 
+    @skipUnlessDBFeature("has_IsEmpty_function")
+    def test_isempty_geometry_null(self):
+        nowhere = State.objects.create(name="Nowhere", poly=None)
+        self.assertSequenceEqual(
+            State.objects.annotate(isempty=functions.IsEmpty("poly")).filter(
+                isempty=None
+            ),
+            [nowhere],
+        )
+        self.assertSequenceEqual(
+            State.objects.annotate(isempty=functions.IsEmpty("poly"))
+            .filter(isempty=False)
+            .order_by("name")
+            .values_list("name", flat=True),
+            ["Colorado", "Kansas"],
+        )
+        self.assertSequenceEqual(
+            State.objects.annotate(isempty=functions.IsEmpty("poly")).filter(
+                isempty=True
+            ),
+            [],
+        )
```

Comment:
We could move the queryset to a separate variable, e.g.
```suggestion
        qs = State.objects.annotate(isempty=functions.IsEmpty("poly"))
        self.assertSequenceEqual(qs.filter(isempty=None), [nowhere])
        self.assertSequenceEqual(
            qs.filter(isempty=False).order_by("name").values_list("name", flat=True),
            ["Colorado", "Kansas"],
        )
        self.assertSequenceEqual(qs.filter(isempty=True), [])
```